src/components/screens/Authenticator/Authenticator.container.vue

```diff
@@ -145,7 +160,7 @@ export default {
         case name === 'SmsCode':
         case name === 'AppCode':
         case name === 'EmailCode':
-          this.handleAuth();
+          this.handleFinishAuthorize();
```

Comment:
Can we merge it with [prev section](https://github.com/endpass/endpass-auth/pull/306/files#diff-e6a549d0f68b3f129bb31930586b9c0bR136)?